api/src/test/java/org/eclipse/microprofile/rest/client/BuilderImpl2.java

```diff
@@ -34,10 +37,31 @@ public RestClientBuilder baseUri(URI uri) {
         throw new IllegalStateException("not implemented");
     }
 
+    @Override
     public RestClientBuilder executorService(ExecutorService executor) {
         throw new IllegalStateException("not implemented");
     }
 
+    @Override
+    public RestClientBuilder sslContext(SSLContext sslContext) {
+        throw new IllegalStateException("not implemented");
+    }
+
+    @Override
+    public RestClientBuilder trustStore(KeyStore trustStore) {
+        throw new IllegalStateException("not implemented");
+    }
+
+    @Override
+    public RestClientBuilder keyStore(KeyStore keyStore, String keystorePassword) {
+        return null;
```

Comment:
Any reason this returns `null` vs `IllegalStateException`?

Reply:
a mistake, I will change it to the exception